apps/updatenotification/lib/Controller/APIController.php

```diff
@@ -39,6 +39,9 @@ class APIController extends OCSController {
 		'bruteforcesettings' => 25,
 		'suspicious_login' => 25,
 		'twofactor_totp' => 25,
+		'files_downloadlimit' => 29,
+		'twofactor_nextcloud_notification' => 30,
+		'app_api' => 30,
```

Comment:
```suggestion
		'app_api' => 30,
		'bruteforcesettings' => 25,
		'circles' => 22,
		'files_downloadlimit' => 29,
		'suspicious_login' => 25,
		'twofactor_nextcloud_notification' => 30,
		'twofactor_totp' => 25,
```

+circles? (though I guess it may not matter at this point)
sort maybe too since we're here?

Reply:
Doesn't help to add things from the past which we don't release anymore.
Also it's sorted by version added atm, just got mixed as app_api is not 31 bug 30 and I amended the fix

Reply:
its easier to compare with the packaging script this way :see_no_evil: